### DISCUSSION TITLE
[需求] 获取正文的功能

### DISCUSSION THREAD

**Discussion by SnowGuest** [imzbf/md-editor-v3#228] [Ideas]
大佬有考虑过出

获取文本正文，跳过 标题 行内代码 等文案 
只获取内容 比如 

# 标题1 
通过 `npm/yarn`  作为包管理 

获取正文变成 “通过 npm/yarn 作为包管理”
这样的场景是 可以作为某些列表的缩略文案 

![image](https://user-images.githubusercontent.com/59474029/224887016-7d599488-b629-435a-b095-9d4427aee949.png)

**Comment by imzbf** [comment]
逻辑比较自由一些，我的第一反应是提取段落的内容，但是也许有的需要引用的内容等等，所以这个最好还是你在保存文章的时候做比较合适。

**Comment by SnowGuest** [comment]
> 

我想的也是在保存文章时候的做，所以这方面我想的是用正则去除特殊标记第一步过滤，然后在生成html，直接获取余下所有的P标签，然后生成基础摘要🤔，或许我可以尝试实现，然后pr

**Comment by imzbf** [comment]
> > 
> 
> 我想的也是在保存文章时候的做，所以这方面我想的是用正则去除特殊标记第一步过滤，然后在生成html，直接获取余下所有的P标签，然后生成基础摘要🤔，或许我可以尝试实现，然后pr

很高兴你愿意参与贡献，只是这个需求过于偏向业务了，不计划在编辑器组件中集成它。或许你可以做成一个库，然后在这个论题下描述你的库功能，以提供有相关需求的小伙伴使用😀。它可能不只是针对这个markdown编辑组件，还可以结合其他的优秀组件使用

**Comment by SnowGuest** [comment]
> > > 
> > 
> > 
> > 我想的也是在保存文章时候的做，所以这方面我想的是用正则去除特殊标记第一步过滤，然后在生成html，直接获取余下所有的P标签，然后生成基础摘要🤔，或许我可以尝试实现，然后pr
> 
> 很高兴你愿意参与贡献，只是这个需求过于偏向业务了，不计划在编辑器组件中集成它。或许你可以做成一个库，然后在这个论题下描述你的库功能，以提供有相关需求的小伙伴使用😀。它可能不只是针对这个markdown编辑组件，还可以结合其他的优秀组件使用

OK感谢解答
